adl/release.adl

```diff
@@ -11,6 +11,10 @@ struct ReleaseConfig
     String prestartCommand;
     String startCommand;
     String stopCommand;
+
+    // Files inside the release to be injected into
+    // the template at release time.
```

Comment:
Is this an absolute path from the release root? Worth updating comment with an example path

I find the comment in the ADL useful when trying to setup and get c2 working :)

Reply:
I've added a comment